brambling/management/commands/send_daily_emails.py

```diff
@@ -0,0 +1,41 @@
+import datetime
+import traceback
+
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from brambling.mail import DailyDigestMailer
+from brambling.models import Person
+
+
+class Command(BaseCommand):
+    def get_recipients(self):
+        return Person.objects.filter(
+            notify_new_purchases=Person.NOTIFY_DAILY,
```

Comment:
Do we think there should be a filter here for `last_new_purchases_digest_sent`? I sort of trust our code more than I trust cron, and it would let us safely run the script as often as needed without overnotifying people.